source/nodejs/adaptivecards/package.json

```diff
@@ -63,5 +63,9 @@
 			"jsx",
 			"json"
 		]
+	},
+	"dependencies": {
+		"dts": "^0.1.1",
```

Comment:
Should these be dev dependencies instead?

Reply:
hmm this likely doesn't need to be here at all ; It's already listed as a devDependency in our root package.json; A stray remnant of the forced npm install earlier when chasing the red herring error.